### DISCUSSION TITLE
Please Remove my Custom Domain Name

### DISCUSSION THREAD

**Discussion by Z-Guy** [community/community#27821] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, I messed up with my custom domain name but it won't be removed when I try to do it. Now I just want to go back to the original name without any customization. Can someone help me with that?

**Comment by mpboom** [comment, marked as answer]
Sure! Please see this page:
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#removing-a-custom-domain

**Comment by Tofutush** [comment]
I did the exact same thing, but it still gets redirected. I was wondering why?

**Reply by mpboom** [reply]
Did you consider browser cache or a delay before DNS updates get reflected?

**Reply by shaGgy3** [reply]
yes that might be true. Because when I open with Crome instead of Safari , its working. Do you know how long it take ?
